README.md

```diff
@@ -207,16 +207,16 @@ blazepack start --verbose > blazepack.log
 ```
 ### Contributing To This Project
 
-```
 Fork this repository to make your own copy
+```
 git clone https://github.com/yourusername/blazepack
 cd blazepack
 npm install
-npm link 
-# With this, blazepack project is added to global packages, 
-# i.e we can make changes to our local blazepack project and see the changes without having to rebuild the package
-# Make any changes to the project and instantly see the changes via blazepack <command> <package>
+mkdir playground # Creates a git-ignored playground folder 
+cd playground
+node ../bin/blazepack.js <command> <name> <template> # Executes our local blazepack project in playground folder
 ```
```

Comment:
Hey @ameerthehacker, I not sure how to frame the sentences exactly in this context, please make necessary changes.